packages/commonwealth/client/styles/components/react_quill/react_quill_editor.scss

```diff
@@ -56,12 +57,14 @@
     }
 
     .CustomQuillFooter {
+      position: sticky;
```

Comment:
why those changes in this file are necessary and what they do? Again, this affect generic component that is used in many places in the app. We should be very cautious with his type of changes.

Reply:
@masvelio I added these because the editor usually grows with the content that gets put in it, but inside a modal it doesn't, and the text was running over the footer. If you want me to move this to a different place or take it out entirely let me know.